torch/quantization/fx/quantize.py

```diff
@@ -18,6 +18,12 @@
 from ..quantization_mappings import (
     get_qat_module_mappings,
 )
+from ..custom_module_class_mappings import (
+    is_custom_module_class,
+    get_observed_custom_module_class,
+    mark_observed_custom_module,
+    is_observed_custom_module,
+)
```

Comment:
Can we split these into separate lines -- it has no performance impact, and in case of the refactoring it's easier to reloacte

Reply:
you mean the import? it's already in separate lines